### DISCUSSION TITLE
Can't resolve 'echarts'

### DISCUSSION THREAD

**Discussion by Ruizhiya** [zouyaoji/vue-cesium#199] [Q&A] [answered]
**ERROR  Failed to compile with 1 error              
 error  in ./node_modules/vue-cesium/es/components/overlays/echarts/index.mjs

Module not found: Error: Can't resolve 'echarts'** 

`ERROR  Failed to compile with 1 error              
 error  in ./node_modules/vue-cesium/es/components/overlays/echarts/index.mjs

Module not found: Error: Can't resolve 'echarts'`

**Comment by zouyaoji** [comment, marked as answer]
This is a known bug, please install the echarts dependency manually and fix it in the next version. npm install echarts --save

**Reply by Ruizhiya** [reply]
thanks
